client/src/Components/Pages/Search/index.js

```diff
@@ -0,0 +1,220 @@
+      this.setState({
+        data: organizations.data,
+        loaded: true
+      });
+    });
+  }
+
+  // functions for autosuggest component
+  // create suggestions array based on user input
+  getSuggestions = (value, organisationsArray) => {
+    const inputValue = value.trim().toLowerCase();
+    const inputLength = inputValue.length;
+    const suggestions = organisationsArray.filter(
+      orga => orga.name.toLowerCase().slice(0, inputLength) === inputValue
+    );
+
+    // in case there are no suggestions still return a value enabling the 'add' box to be rendered
+    if (suggestions.length === 0) {
+      return [{ isEmpty: true }];
+    }
+    return suggestions;
+  };
+
+  // gets called when suggestions gets clicked
+  getSuggestionValue = suggestion => suggestion.name;
+
+  // Autosuggest will call this function every time suggestions are updated
+  onSuggestionsFetchRequested = ({ value }) => {
+    this.setState({
+      suggestions: this.getSuggestions(value, this.state.data)
+    });
+  };
+
+  // Autosuggest will call this function every time you need to clear suggestions.
+  onSuggestionsClearRequested = () => {
+    this.setState({ suggestions: [] });
+  };
+
+  onChange = (event, { newValue }) => {
+    this.setState({ value: newValue });
+  };
+
+  // render functions
+
+  // renders individual suggestions in autosuggest search section
+  renderSuggestion = suggestion => {
+    // check if no suggestion available and return so that renderSuggestionsContainer function is still being called (gets deactivated otherwise)
+    if (suggestion.isEmpty) {
+      return;
+    } else {
+      return (
+        <ProfileLink to={`/profile/${suggestion._id}`}>
+          <SuggestionBox orgType={suggestion.category}>
+            <InnerDivSuggestions>
+              <SymbolDiv>
+                {SVGCreator("mobile-search-icon")}
+                {SVGCreator(`${organizationIcons[suggestion.category].symbol}`)}
+              </SymbolDiv>
+              <OrganisationDetailsDiv>
+                <h3>{suggestion.name}</h3>
+                <ReviewDetailsDiv>
+                  {StarRateCreator(suggestion)}
+                  <p>{suggestion.totalReviews} reviews</p>
+                </ReviewDetailsDiv>
+              </OrganisationDetailsDiv>
+              <ArrowDiv>
+                {SVGCreator(`${organizationIcons[suggestion.category].arrow}`)}
+              </ArrowDiv>
+            </InnerDivSuggestions>
+          </SuggestionBox>
+        </ProfileLink>
+      );
+    }
+  };
+
+  // renders all elements and the add item footer
+  renderSuggestionsContainer = ({ containerProps, children, query }) => (
+    <div {...containerProps}>
+      {children}
+      {
+        <div className="footer">
+          <ProfileLink to="">
+            <AddItemBox>
+              <InnerDivSuggestions>
+                <SymbolDiv>{SVGCreator("add-item-icon")}</SymbolDiv>
+                <OrganisationDetailsDiv mt="0.2rem">
```

Comment:
can we add this styling to the styled-component? its good to keep things consistent and all styling in the same place if possible